dotnetv3/S3/LifecycleExample/LifecycleExample/Lifecycle.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+namespace LifecycleExample
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Amazon;
+    using Amazon.S3;
+    using Amazon.S3.Model;
+
+    /// <summary>
+    /// This example shows how to work with Amazon Simple Storage Service
+    /// (Amazon S3) bucket lifecycle settings. It was created with the AWS SDK
+    /// for .NET version 3.7 and .NET Core 5.0.
+    /// </summary>
+    public class Lifecycle
+    {
+        public static async Task Main()
+        {
+            // If the Amazon Region associated with the default user
+            // is different from the region where the bucket is located,
+            // pass the bucket region to the AmaS3Client constructor.
+            // The parameter should look like this:
+            //      RegionEndpoint.USWest2
+            var client = new AmazonS3Client();
+            const string BucketName = "doc-example-bucket";
+
+            await AddUpdateDeleteLifecycleConfigAsync(client, BucketName);
+        }
+
+        /// <summary>
+        /// This method creates, adds, and then removes lifecycle information
+        /// to the S3 bucket named in the bucketName parameter.
```

Comment:
S3 bucket -> Amazon S3 bucket

Reply:
Third time's the charm. It should be just S3.